doc/howto/images_manage.md

```diff
@@ -67,6 +67,16 @@ To edit the full image properties, including the top-level properties, enter the
 
     lxc image edit <image_ID>
 
+## Delete an image
+To delete an image on a server, enter the following command:
```

Comment:
This is a little ambiguous - what is "a server"? This could be misunderstood as "a remote server", but you actually mean the local server.
So maybe leave it out (just "To delete an image, enter ...") or be specific that we talk about a local image ("To delete a local copy of an image, enter ...").

Reply:
Also, for consistency, can you add an empty line after the heading?